### PR TITLE
Jungle

### PR DESCRIPTION
**Pull request by Haochenlucas**
Jungle feature finished

### CODE REVIEW

File: GamingSupervisor/GamingSupervisor/GameStateIntegration.cs [hunk 1]
```diff
@@ -165,7 +165,7 @@ public string SteamID
         }
 
         private readonly object gameTimeLock = new object();
-        private int gameTime;
+        public int gameTime;
```
Comment (theoculus8) [inline GamingSupervisor/GamingSupervisor/GameStateIntegration.cs:168]: This should not be made public. There is a thread safe getter right below this called "GameTime".

File: GamingSupervisor/GamingSupervisor/ReplayAnalyzer.cs [hunk 2]
```diff
@@ -326,6 +336,8 @@ private void HandleHighlight()
 
         private void HandleGamePlay()
         {
+            #region Hero introduction
```
Comment (theoculus8) [inline GamingSupervisor/GamingSupervisor/ReplayAnalyzer.cs:339]: I don't think we should be using regions. If code should be divided up, it should be put into functions.

File: GamingSupervisor/GamingSupervisor/GameStateIntegration.cs (hunk 1 above)
Comment (Haochenlucas) [reply GamingSupervisor/GamingSupervisor/GameStateIntegration.cs:168]: I will change that back.

File: GamingSupervisor/GamingSupervisor/ReplayAnalyzer.cs (hunk 2 above)
Comment (Haochenlucas) [reply GamingSupervisor/GamingSupervisor/ReplayAnalyzer.cs:339]: Yeah, you are right. I will make a function.

**Review by yatzyt — APPROVED** [review]

**Review by dominate0704 — APPROVED** [review]

**Pull request closed without merging**